posix.mak

```diff
@@ -269,12 +269,10 @@ dlangspec.tex : $(DMD) $(DDOC) latex.ddoc dlangspec-consolidated.d
 	$(DMD) -Df$@ $(DDOC) latex.ddoc dlangspec-consolidated.d
 
 # Run twice to fix multipage tables and \ref uses
-dlangspec.dvi : dlangspec.tex
-	latex $^
-	latex $^
-
-$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.dvi
-	dvipdf $^ $@
+$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.tex
+	pdflatex -draftmode $^
+	pdflatex $^
+	-mv dlangspec.pdf $(DOC_OUTPUT_DIR)/dlangspec.pdf
```

Comment:
Using `/tmp` is not ideal because it is a global resource - junk created by one user can get in the way of another's. I think it should be only used with `mkstemp` or the like, which is not available from Makefiles, so I'll use a local `.tmp` instead.